dub.sdl

```diff
@@ -7,7 +7,7 @@ libs "ssl" "crypto" platform="posix"
 configuration "library-autodetect" {
 	targetType "sourceLibrary"
 	excludedSourceFiles "source/deimos/openssl/applink.d"
-	preGenerateCommands `${DUB} scripts/generate_version.d` platform="posix"
+	preGenerateCommands `$DC -run scripts/generate_version.d` platform="posix"
```

Comment:
I've just seen that GDC (v11) doesn't support `-run`. @ibuclaw: Is there a similar way?

Reply:
Wouldn't `-run` be in `gdmd`?

Reply:
It is, yes - but dub prefers `gdc` over `gdmd` AFAICT.